examples/large_models/vllm/Readme.md

```diff
@@ -0,0 +1,7 @@
+# Example showing inference with vLLM
+
+This folder contains multiple demonstrations showcasing the integration of [vLLM](https://github.com/vllm-project/vllm) with TorchServe, running inference on `mistralai/Mistral-7B-v0.1` model and multiple LoRA models.
```

Comment:
@lxning There is already a vLLM / Mistral example added by PR: https://github.com/pytorch/serve/pull/2781/ if you 
It will be good to combine with that if you plan to Mistral as the base model. 

The LoRA example should lighlight loading of the lora adapter weights

cc: @ankithagunapal @mreso

Reply:
yes, this PR will combine Mistral example. Eventually each example has different model-config.yaml.